libsupport/include/katana/EntityTypeManager.h

```diff
@@ -248,7 +248,7 @@ class KATANA_EXPORT EntityTypeManager {
             ErrorCode::NotFound, "Type {} does not exist", name);
       }
     }
-    return res;
+    return Result<SetOfEntityTypeIDs>(res);
```

Comment:
@srogatch : We do have `MakeResult(val)` now that I added for such scenarios. Haven't tested it with CUDA compiler.

Reply:
It works with CUDA, though it gives a weird error if you forget `std::move()`. I'm committing it.